helix-core/src/main/java/org/apache/helix/spectator/RoutingDataCache.java

```diff
@@ -130,23 +130,21 @@ public synchronized void refresh(HelixDataAccessor accessor) {
         }
         _propertyDataChangedMap.put(HelixConstants.ChangeType.CUSTOMIZED_VIEW, false);
       }
-        break;
-      default:
-        break;
+      break;
+        default:
```

Comment:
nit : can remove this?

Reply:
Thanks for the suggestion. This is an accidental change and i changed it back. Let's keep this PR only for log formatting.